pkg/ebpf/tracer.go

```diff
@@ -435,8 +435,8 @@ func (t *Tracer) getEbpfTelemetry() map[string]int64 {
 	telemetry := &kernelTelemetry{}
 	if err := t.m.LookupElement(mp, unsafe.Pointer(&zero), unsafe.Pointer(telemetry)); err != nil {
 		// This can happen if we haven't initialized the telemetry object yet
-		// so let's just use a debug log
-		log.Debugf("error retrieving the telemetry struct: %s", err)
+		// so let's just use a trace log
+		log.Tracef("error retrieving the telemetry struct: %s", err)
```

Comment:
any reason this changed?

Reply:
👍  it was a bit noisy until the telemetry was initialized